libqtile/backend/base.py

```diff
@@ -253,6 +250,54 @@ def info(self) -> dict[str, Any]:
         """
         return {}
 
+    @expose_command()
+    def keep_above(self, enable: bool | None = None, include_internal: bool = False):
+        """Keep this window above all others"""
+
+    @expose_command()
+    def keep_below(self, enable: bool | None = None, include_internal: bool = False):
+        """Keep this window below all others"""
```

Comment:
What does `include_internal` do? Perhaps worth adding a line to describe that, like with the methods below?

Reply:
We stack windows in the current group but, as the bar's group is `None`, we don't stack it directly. Using `include_internal` adds the screen's bars to the list of windows to stack. Typically, we don't want them included because otherwise things like `move_up/down` will include the bar.

However, it's only the bar that calls `change_layer` with `include_internal=True` but I added some code that exits `change_layer` when the window's `group` is `None`. 

We can probably strip out some of that code.

The bars are still included when doing that check I mentioned earlier (checking layers are in the correct order)

Reply:
ok makes sense :+1: